reference/v8.json

```diff
@@ -823,13 +897,19 @@
       "function": "piecewise-constant",
       "zoom-function": true,
       "property-function": true,
-      "values": [
-        "map",
-        "viewport",
-        "auto"
-      ],
+      "values": {
+        "map": {
+            "doc": "When `symbol-placement` is set to `point`, produces glyphs whose x-axes are aligned east-west. When set to `line`, produces glyphs whose x-axes are aligned with the line."
```

Comment:
>  produces glyphs whose x-axes are aligned east-west

"produces glyphs who x-axes" is unnecessarily techincal. This could benefit from plainer language. 

Point: Aligns text east-west. 
Line: Places text along corresponding line geometry.